app/components/main/main.component.ts

```diff
@@ -158,7 +168,7 @@ export class MainComponent implements OnInit, MenuEventListener, REPLDelegate, T
     menuEvent(e: MenuEvent) {
         switch (e) {
             case MenuEvent.NEW_FILE:
-                this.newFile();
+                this.promptNewFile();
```

Comment:
Not sure why this isn't triggering for the first tab, but it should show this prompt at startup, or at least a blank screen with a message say please create a new document or load something.